src/InfluxDB2/WriteApi.php

```diff
@@ -38,13 +38,50 @@ public function __construct(array $writeOptions = null)
     }
 }
 
+class PointSettings
+{
+    private $defaultTags;
+
+    public function __construct(array $defaultTags = null)
+    {
+        $this->defaultTags = array();
+
+        if (!empty($defaultTags)) {
```

Comment:
```php
if (!empty($defaultTags)) {
             foreach (array_keys($defaultTags) as $key) {
                 $this->addDefaultTag($key, $defaultTags[$key]);
             }
         }
```

is useless, because evaluation of tags is done in `\InfluxDB2\WriteApi::addDefaultTags`